exercises/crypto-square/test/crypto_square_tests.erl

```diff
@@ -0,0 +1,33 @@
+%% based on canonical data version 3.2.0
+%% https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/crypto-square/canonical-data.json
+
+-module(crypto_square_tests).
+
+-define(TESTED_MODULE, (sut(crypto_square))).
+-define(TEST_VERSION, 1).
```

Comment:
Those `-define`s aren't needed anymore since the introduction of the `erl_exercism` library.

`TEST_VERSION` is a nice helper though that makes it easier to bump it.

`TESTED_MODULE` is not used throughout the tests at all. `sut/1` isn't defined anymore. The only reason it does not break is because we do not use the macro anywhere.

`erl_exercism` today works through a parse transform. It basically looks up the modulename from the tests, strips `_tests` from it, and searches for occurences through the testsmodule and replaces them with calls to the `example` module if `src/example.erl` exists.

For a student that is much easier to read than the old construct.

Reply:
In that case, the passage needs to be removed from the contribution guidelines ;)

Reply:
Good point ;)